src/core/SEO/Homepage/structuredData.tsx

```diff
@@ -0,0 +1,18 @@
+import { searchUrl } from "../../../components/App/routes"
+
+export const structuredData = (shop) => {
+    let data = {
```

Comment:
Why let?
IMO just directly return JSON.stringify({ ... })

Reply:
This is still WIP sorry for missing label